app.rb

```diff
@@ -0,0 +1,80 @@
+# HARDER User Stories:
+# If signed in, don't give "Sign Up" or "Sign In" options
+# Make sure paths flow and edit CSS
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative './lib/user.rb'
+require_relative './database_connection.rb'
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+  register Sinatra::Flash
+
+  before do
+    @user = User.find(session[:user_id]) unless session[:user_id].nil?
+  end
+
+  get '/' do
+    erb :index
+  end
+
+  get '/peeps' do
+    @peeps = Peep.all.order(created_at: :desc)
+    erb :'peeps/index'
+  end
+
+  get '/peeps/new' do
+    erb :'peeps/new'
+  end
+
+  post '/save_peep' do
+    peep = @user.peeps.create(params[:peep])
```

Comment:
Rather than creating the peep through `@user.peeps` could we do something like `Peep.create` here?

Reply:
I did it this way because I read that this is a bit of ActiveRecord 'sugar' that means a peep will be created with the user_id column filled in with @user.id.